traffic_portal/app/src/common/api/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = angular.module('trafficPortal.api', [])
     .service('cdnService', require('./CDNService'))
     .service('changeLogService', require('./ChangeLogService'))
     .service('coordinateService', require('./CoordinateService'))
+    .service('deliveryServiceConsistentHashRegexService', require('./DeliveryServiceConsistentHashRegexService'))
```

Comment:
what do you think about just putting this in the deliveryServiceService? and getting rid of deliveryServiceConsistentHashRegexService.

Reply:
that might make more sense because the consistent hash regex is related to a Delivery Service and but it's not its own object... let me shuffle some code around and see how it works